.travis.yml

```diff
@@ -29,81 +29,34 @@ matrix:
     - env: allowTestFail=true
 
   include:
-    # MacOS Builds
-    - name: "OSX GCC"
-      os: osx
-      compiler: gcc
-      before_install: skip # Only for Linux
-      before_script:
-        - mkdir build && cd build && cmake .. -DBUILD_TEST=TRUE
-      script:
-        - make
-        - ./tst/producerTest
-
-    - name: "OSX Clang"
-      os: osx
-      compiler: clang
-      before_install: skip # Only for Linux
-      before_script:
-        - mkdir build && cd build && cmake .. -DBUILD_TEST=TRUE -DCOMPILER_WARNINGS=TRUE
-      script:
-        - make
-        - ./tst/producerTest
-
-    # Code Coverage
-    - name: "Linux GCC Code Coverage"
-      os: linux
-      compiler: gcc
-      before_script:
-        - mkdir build && cd build && cmake .. -DCODE_COVERAGE=TRUE -DBUILD_TEST=TRUE -DBUILD_GSTREAMER_PLUGIN=TRUE -DBUILD_JNI=TRUE
-      after_success:
-        - for test_file in $(find CMakeFiles/KinesisVideoProducer.dir gstkvssink.dir KinesisVideoProducerJNI.dir -name '*.gcno'); do gcov $test_file; done
-        - bash <(curl -s https://codecov.io/bash)
-
-    # AddressSanitizer
-    - name: "Linux Clang AddressSanitizer"
-      os: linux
-      compiler: clang
-      env: ASAN_OPTIONS=detect_odr_violation=0:detect_leaks=1
-      before_script: mkdir build && cd build && cmake .. -DBUILD_TEST=TRUE -DADDRESS_SANITIZER=TRUE -DBUILD_GSTREAMER_PLUGIN=TRUE -DBUILD_JNI=TRUE
 
     # UndefinedBehaviorSanitizer
-    - name: "Linux Clang UndefinedBehaviorSanitizer"
-      os: linux
-      compiler: clang
-      env: allowTestFail=true
-      # env: UBSAN_OPTIONS=halt_on_error=1
-      before_script: mkdir build && cd build && cmake .. -DBUILD_TEST=TRUE -DUNDEFINED_BEHAVIOR_SANITIZER=TRUE -DBUILD_GSTREAMER_PLUGIN=TRUE -DBUILD_JNI=TRUE
+    # - name: "Linux Clang UndefinedBehaviorSanitizer"
```

Comment:
Let us uncomment all the sanitizers on travis.

Reply:
Uncommented in the latest commit